log/sequencer.go

```diff
@@ -241,8 +236,12 @@ func (s Sequencer) initMerkleTreeFromStorage(ctx context.Context, currentRoot *t
 		return compact.NewTree(s.hasher), nil
 	}
 
-	// Initialize the compact tree state to match the latest root in the database
-	return s.buildMerkleTreeFromStorageAtRoot(ctx, currentRoot, tx)
+	// Initialize the compact tree state to match the latest root in the database.
+	mt, err := s.buildMerkleTreeFromStorageAtRoot(ctx, currentRoot, tx)
+	if err != nil {
```

Comment:
This is going to spam the log if something goes wrong. Unfortunately the glog api doesn't have rate limiting. The failed signing run metric might be enough.

Maybe leave it in for now and TODO it for review later.

Reply:
This PR doesn't change errors or `glog` behaviour, so out of scope. The amount of spam is (as before) one line per failed signing run (written somewhere up the stack),  which I think is reasonable.

Reply:
Is it the same? It looked to be adding extra logging. 

To be clear I was just asking for a TODO, not to change glog or anything else.

Reply:
No extra logging: Ctrl+F "glog." finds nothing. Changes in `sequencer.go` are mostly about indentation: before the code was in a callback, now it's not.

Reply:
It's not clear from the diffs. It's just moved around from one func to another? OK then.

Reply:
Yes. Github diff is not great here.